src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  &--4 {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
Why do these properties apply only for `stars--4`, if we have modificator with only 3 active starts it will broke everything